src/CoreBundle/private/modules/MTTableManager/MTTableManager.class.php

```diff
@@ -340,14 +341,58 @@ protected function AddURLHistory()
         }
 
         // If list was opened within a frame, url should not be added to history. The parameter field "sRestrictionField" can be used as an indicator.
-        if (array_key_exists('sRestriction', $params) && array_key_exists('sRestrictionField', $params)) {
-            $preventHistoryEntry = true;
+        if (false === $this->isInFrame()) {
+
+            $tableName = $this->oTableConf->GetName();
+
+            $iconCssClass = trim($this->getIconCssClassForTable($this->oTableConf->id));
+
+            if ('' !== $iconCssClass) {
+                $tableName = '<i class="'.TGlobal::OutHTML($iconCssClass).'"></i> '.$tableName;
+            }
+
+            $breadcrumb = $this->getBreadcrumbService()->getBreadcrumb();
+            $breadcrumb->AddItem($params, $tableName);
         }
+    }
 
-        $sTableName = strip_tags(preg_replace('/(\r\n|\n|\r)/', ' ', $this->oTableConf->sqlData['translation']));
-        if (!$preventHistoryEntry) {
-            $this->global->GetURLHistory()->AddItem($params, $sTableName);
+    private function getIconCssClassForTable(string $tableId): string
+    {
+        $menuItem = \TdbCmsMenuItem::GetNewInstance();
+        if (false === $menuItem->LoadFromFields(array(
+            'target' => $tableId,
+            'target_table_name' => 'cms_tbl_conf')
+        )) {
+            return '';
         }
+
+        return $menuItem->fieldIconFontCssClass;
+    }
+
+    protected function isInFrame()
```

Comment:
```suggestion
    private function isInFrame(): bool
```

Or what is the intention behind making the method protected?

Reply:
Good, that you found this one. There was a reason for the protected type. The extension MTTableManagerComponent is always used in a frame situation, so here i want to extend the method and always return true. Added this.